casper/src/test/scala/coop/rchain/casper/helper/DeployerIdContract.scala

```diff
@@ -0,0 +1,27 @@
+package coop.rchain.casper.helper
+import cats.effect.Concurrent
+import coop.rchain.models.rholang.implicits._
+import coop.rchain.models.{GDeployerId, ListParWithRandom, Par}
+import coop.rchain.rholang.interpreter.Runtime.SystemProcess
+import coop.rchain.rholang.interpreter.{ContractCall, RhoType}
+import coop.rchain.shared.ByteStringOps._
+
+object DeployerIdContract {
```

Comment:
We should probably add a fat warning saying this should under no circumstances be available in production :)

Reply:
Done